### DISCUSSION TITLE
Deploy topic Operator in different namespace

### DISCUSSION THREAD

**Discussion by oesebus** [strimzi/strimzi-kafka-operator#9400] [Q&A]
Hi , 

I would like to know what would be the best approach to create topics in namespace B even if kafka and operators are deployed in Namespace A . 

I see in the documentation its possible to deploy multiple operators but i don't know how to handle authorizations and isolation in a efficient way with such design then . 

Many thanks for your help 

Ismael Idrissi

**Comment by scholzj** [comment]
In the `Kafka` CR, you can configure the namespace that will be watched by the Topic Operator. It is always one namespace, but it can be a different namespace from where the operator runs. Look in the docs for the `watchedNamespace` field.
